src/karmen_backend/server/__init__.py

```diff
@@ -22,7 +22,7 @@ def setup_celery(flask_app):
         app.config["REDIS_PORT"],
     )
     celery_inst = Celery(
-        flask_app.import_name, backend=redis_backend, broker=redis_backend,
+        flask_app.import_name, backend=redis_backend, broker=redis_backend
```

Comment:
Me to but it looks like [black](https://github.com/psf/black#trailing-commas) doesn't. And since this is the industry standard formatter for some time now, I wouldn't even try to make change his opinion on that :)